pkg/endpoints/router.go

```diff
@@ -52,8 +52,9 @@ func (r Resource) RegisterEndpoints(container *restful.Container) {
 		Consumes(restful.MIME_JSON).
 		Produces(restful.MIME_JSON)
 
-	logging.Log.Info("Adding v1, and API for pipelines")
+	logging.Log.Info("Adding v1, and API for k8s resources and pipelines")
 
+	wsv1.Route(wsv1.GET("/").To(r.getAllNamespaces))
```

Comment:
Document this new endpoint under DEVELOPMENT.md please 😄 I know it's not really going to be used much by end users but at least we'll all know it's exist and what its contract is